### PR TITLE
Adding Video recordings to the tutorials

### PR DESCRIPTION
**Pull request by subinamehta**

### CODE REVIEW

**Comment by shiltemann** [comment]
@subinamehta awesome, thanks for the videos! And for doing good captions :+1:

File: topics/proteomics/tutorials/clinical-mp-1-database-generation/tutorial.md [hunk 1]
```diff
@@ -106,7 +116,7 @@ The first workflow for the clinical metaproteomics data analysis is the Database
 >
 > 1. **Import the workflow** into Galaxy:
 >
->    {% snippet faqs/galaxy/workflows_run_trs.md path="topics/proteomics/tutorials/clinical-mp-database-generation/workflows/main_workflow.ga" title="Pretreatments" %}
+>    https://usegalaxy.eu/u/galaxyp/w/wf1databasegenerationworkflow
```
[the remark this answers is not in the document]
Comment (subinamehta) [reply topics/proteomics/tutorials/clinical-mp-1-database-generation/tutorial.md:119]: I tried doing that but the lint test kept failing, could you please help me with that?

[the remark this answers is not in the document]
Comment (shiltemann) [reply topics/proteomics/tutorials/clinical-mp-1-database-generation/tutorial.md:119]: ok  no worries, I can take care of this :+1:

**Comment by subinamehta** [comment]
Thank you so much 🙏🏻

*Subina Mehta*
Researcher, Dept. of Biochemistry, Molecular Biology and Biophysics
Bioinformatics Specialist, Proteogenomics Shared Resource, Masonic Cancer
Center
University of Minnesota
7-166 MCB
420 Washington Ave SE
Minneapolis, MN 55455

Lab:     612-624-0381
Phone: 612-500-8841

Email: ***@***.*** ***@***.***>*
*www.galaxyp.org* <http://www.galaxyp.org/>
http://z.umn.edu/itcrgalaxyvideo

**Comment by shiltemann** [comment]
Hey @subinamehta, I will merge this now, but it would be great if at some point you could add some workflow tests. This makes it easy for us to test which Galaxy servers support your tutorials.

There are instructions here: https://training.galaxyproject.org/training-material/faqs/gtn/gtn_workflow_testing.html

**Review by shiltemann — APPROVED** [review]

**Pull request merged**